spec/requests/maintenance_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe '[Maintenance mode]', type: :request do
+  it 'Maintenance mode can be switched on and off' do
+    login(dont_handle_splash: true)
```

Comment:
Actually, it looks like login is necessary even to access the splash page in siteminder auth mode.

When I don't login, the test fails with:
```
  1) [Maintenance mode] Maintenance mode can be switched on and off
     Failure/Error: visit root_path
     
     ActionController::RoutingError:
       No route matches [GET] "/logout"
```